template/python3-armhf/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM armhf/python:3.6-alpine
```

Comment:
I'm not sure that the prefix is needed of `armhf` anymore since the mutli-arch builds are now official.

Reply:
(hmm.. `python:3.6-alpine` doesn't exist, but `armhf/python:3.6-alpine` does.. we should check the library via @tiannon to see where the Python 3 / Alpine image is for ARM )